addons/mail/sfu_server/README.md

```diff
@@ -0,0 +1,81 @@
+# Odoo: Discuss SFU
+
+## Overview
+
+Contains the code for an SFU (Selective Forwarding Unit) server 
+to be used alongside the Odoo Discuss module. The SFU server is responsible for
+handling the WebRTC connections between users in a call and relies on an Odoo
+master server as the source of truth for the channels and rtc sessions.
+
+## Prerequisites
+
+- An Odoo server with the Discuss module installed
+- [Node.js](https://nodejs.org/)
+
+## Deployment
+
+1. Install dependencies.
+    ```bash
+        npm install
+    ```
+2. Run the build script to generate the client bundle (requires dev dependencies).
+    ```bash
+        npm run build 
+    ```
+   
+   or
+   ```bash
+        npm run build:dev # for development with source maps
+    ```
+   
+3. Run the server (see `config.js` for env variables).
+    ```bash
+        # the PUBLIC_IP env variable is mandatory, it is necessary to establish the RTC connections
+        npm PUBLIC_IP=134.123.222.111 ODOO_URL=https://www.odoo.com/ run start
```

Comment:
Does that mean that the `ODOO_URL` is the source of truth for **all** the sessions? (like all sessions in all databases of the SaaS / PaaS)

Reply:
currently, it is only used for 2 things:

1) it is the url that we will request to obtain the public encryption key used to decrypt the JWTs that contain the "identity" of users. Technically the public key provider can be any url, unrelated to odoo.

2) it is used as the cors allowed url for the `/disconnect` route that can force the disconnect of a session, that one could be removed, the payload is still encrypted (jwt) as the origin could be forged anyways.